### PR TITLE
chore(deps): Bump storybook to 6.4 and creevey to 0.8.0

### PR DESCRIPTION
**Pull request by mayank99**
- Updated storybook packages to latest (`6.4.19`).
- Added sb eslint plugin which comes with 6.4
- Updated `creevey` to 0.8.0 now that it's available

## Checklist

- [x] ~Add meaningful unit tests for your component (verify that all lines are covered)~
- [x] Verify that all existing tests pass
- [x] ~Add component features demo in Storybook (different stories)~
- [x] ~Approve test images for new stories~
- [x] ~~Add screenshots of the key elements of the component~~

### CODE REVIEW

**Comment by veekeys** [comment]
Forgot to run new linter and fix that :)

File: .eslintrc.js [hunk 1]
```diff
@@ -17,7 +21,8 @@ module.exports = {
     "plugin:@typescript-eslint/recommended", // Uses the recommended rules from @typescript-eslint/eslint-plugin
     "plugin:react-hooks/recommended",
     "prettier",
-    "plugin:prettier/recommended" // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "plugin:prettier/recommended", // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "plugin:storybook/recommended"
```
Comment (veekeys) [inline .eslintrc.js:25]: So message above recommends, that prettier plugin is the last one. Lets do that, not to risk random issues ?

File: stories/core/ErrorPage.stories.tsx [hunk 2]
```diff
@@ -128,20 +128,20 @@ customHtmlMessage.args = {
...
 
-export const translatedMessages: Story<ErrorPageProps> = (props) => (
+export const TranslatedMessages: Story<ErrorPageProps> = (props) => (
   <ErrorPage {...props} />
 );
-translatedMessages.args = {
+TranslatedMessages.args = {
   errorType: 'generic',
   errorMessage:
     "We can't find the iModel that you are looking for or it does not exist. Visit the iModel HUB or contact our support team.",
```
Comment (gretanausedaite) [inline stories/core/ErrorPage.stories.tsx:147]: Random: why are we using "fOrBiDdEn" as different language? Why not actual language like Spanish or French?

Comment (mayank99) [reply stories/core/ErrorPage.stories.tsx:147]: Ask the person who wrote that code 😄  But I don't want to change that in this PR

File: .eslintrc.js (hunk 1 above)
Comment (mayank99) [reply .eslintrc.js:25]: Done

**Review by gretanausedaite — COMMENTED** [review]
Tried to run this branch locally and got error:
error @typescript-eslint/experimental-utils@5.15.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.1"

**Comment by mayank99** [comment]
> Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.1"

Well, it says right there. You need to update your node version 😄

I recommend using [`nvm`](https://github.com/nvm-sh/nvm) to get the latest 16.14.2

File: stories/core/ErrorPage.stories.tsx (hunk 2 above)
Comment (veekeys) [reply stories/core/ErrorPage.stories.tsx:147]: > > Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.1"
> 
> Well, it says right there. You need to update your node version 😄
> 
> I recommend using [`nvm`](https://github.com/nvm-sh/nvm) to get the latest 16.14.2

But it is weird, that linter package requires certain node version.. And then it also should appear in our documentation somewhere...
So not nice :/

Comment (veekeys) [reply stories/core/ErrorPage.stories.tsx:147]: Oh, sorry.. I dont know how I commented in here versus reply in conversation :DD

**Comment by mayank99** [comment]
> But it is weird, that linter package requires certain node version.. And then it also should appear in our documentation somewhere...

We were talking about moving everything (just github workflows I guess) to Node 16. We already did that in the CSS repo.

**Review by veekeys — APPROVED** [review]

**Review by gretanausedaite — APPROVED** [review]
Storybooks starts.

**Pull request merged**